deploy/compose/docker-compose.yml

```diff
@@ -29,13 +28,9 @@ services:
       - "RUST_LOG=${HOPRD_LOG_LEVEL}"
       - RUST_BACKTRACE=full
       - HOPRD_CONFIGURATION_FILE_PATH=/app/hoprd.cfg.yaml
-      - HOPRD_API_PORT=3001
       # - OPENTELEMETRY_COLLECTOR_URL=http://server.
     volumes:
-      - ./hoprd.cfg.yaml:/app/hoprd.cfg.yaml
-      - ./hopr.id:/app/hopr.id
-      - hoprd_data:/app/hoprd/
-      - '/var/run/docker.sock:/var/run/docker.sock'
```

Comment:
The volume docker.sock is required by nodeexporter

Reply:
Wouldn't this only be needed if hoprd wants to spin up more containers inside itself?
Nodeexporter gets access to the docker.sock anyway. Are you sure this needs to be mounted inside hoprd as well?

Reply:
@ausias-armesto https://hub.docker.com/r/prom/node-exporter does not seem to be needed at all.